app/src/main/java/com/kickstarter/services/KSApolloClient.kt

```diff
@@ -185,6 +188,83 @@ class KSApolloClient(val service: ApolloClient) : ApolloClientType {
         }
     }
 
+    private fun createCommentObject(commentFr: fragment.Comment?): Comment {
+        val author = User.builder()
+            .id(decodeRelayId(commentFr?.author()?.fragments()?.user()?.id()) ?: -1)
+            .name(commentFr?.author()?.fragments()?.user()?.name())
+            .avatar(
+                Avatar.builder()
+                    .medium(commentFr?.author()?.fragments()?.user()?.imageUrl())
+                    .build()
+            )
+            .build()
+
+        return Comment.builder()
+            .id(decodeRelayId(commentFr?.id()) ?: -1)
+            .author(author)
+            .repliesCount(0)
+            .body(commentFr?.body())
+            .authorBadges(listOf())
```

Comment:
author badges is always gonna be and empty list? is not being retrieve on the query?